index.html

```diff
@@ -205,21 +205,19 @@ <h3>Skills</h3>
               </ul>
             </div>
           </section>
-          
-		<section id="contact_form">
-			<div class="contact_detail">
-				<h2>Contact me</h2>
-				<p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
-					I’d love to help with it!
-				</p>
-			</div>
-			<form action="https://formspree.io/f/xzbqlavz" method="POST" class="form">
-					<input class="input" type="text" name="text" maxlength="30" required placeholder="Enter your full name here">
-					<input class="input" type="email" name="email" required placeholder="Enter your email here">
-					<textarea class="input" name="message" placeholder="Enter your message here" maxlength="500" required></textarea>
-				  <button type="submit" class="button" >Get in touch</button>
-			</form>
-		</section>
+          <section class="form-section">
+            <h3>Contact me</h3>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+            <form action="https://formspree.io/f/xzbqlavz" method="post">
+              <input type="text" maxlength="30" placeholder="Full Name" required>
+              <input type="email" name="email" id="" placeholder="Email address" required>
+              <textarea name="message" id="" maxlength="500" placeholder="Write your message here..." required></textarea>
+            <div class="card-text">
+              <button type="submit">Get in touch</button>
```

Comment:
👍 Well done on implementing the necessary styles for an interactive UI on your button as the previous reviewer requested! However, there are still a few changes that need to be made to match the design specifications. It is important to use the proper colors as mentioned in the design. By using the correct colors specified in the design, you will ensure consistency and alignment with the overall visual aesthetics of your project. Keep up the great work! 👍🎨

Reply:
Thank you for pointing this out. I have amended the requirements. In my reckoning being a novice it is a good job done obviously it was not possible without your(all reviewers') efforts. Thank you for being there.